cmd/scheduler/app/options/validation.go

```diff
@@ -0,0 +1,31 @@
+package options
+
+import (
+	"net"
+
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+// Validate checks Options and return a slice of found errs.
+func (opts *Options) Validate() field.ErrorList {
```

Comment:
```suggestion
func (o *Options) Validate() field.ErrorList {
```
Receiver names are different. As we already have a receiver name `o`.

Reply:
done